ProvisioningTool/include/constants.h

```diff
@@ -101,14 +101,16 @@ constexpr char kSeFactoryProvisionLock[] = "se_factory_lock";
 constexpr char kUnLockProvision[] = "unlock_provision";
 
 // Instruction constatnts
-// TODO Modify according to keymint
-constexpr int kAttestationIdsCmd = INS_BEGIN_KM_CMD + 1;
-constexpr int kPresharedSecretCmd = INS_BEGIN_KM_CMD + 2;
-constexpr int kOemLockProvisionCmd = INS_BEGIN_KM_CMD + 3;
```

Comment:
Looks like the HAL changes are missing.

Reply:
Made HAL changes wrt init strong box command